benchmark/benchncnn.cpp

```diff
@@ -241,6 +241,45 @@ static std::vector<ncnn::Mat> parse_shape_list(char* s)
     return mats;
 }
 
+// Helper to check if a model is int8 quantized
+bool is_model_int8(const ncnn::Net& net, const char* model_name)
+{
+    // Check filename convention
+    if (strstr(model_name, "int8") != nullptr)
+        return true;
+
+    // Check actual model layers
+    for (size_t i = 0; i < net.layers().size(); i++)
+    {
+        const ncnn::Layer* layer = net.layers()[i];
+        if (layer->support_int8_storage)
+            return true;
```

Comment:
`support_int8_storage` indicates that this layer supports int8 type input, but it does not mean that the current layer must not use int8 calculation, considering dynamic quantization, such as rnn. However, I think the logic here is OK, as a supplementary judgment